src/helpers.coffee

```diff
@@ -233,3 +233,28 @@ exports.nameWhitespaceCharacter = (string) ->
     when '\r' then 'carriage return'
     when '\t' then 'tab'
     else string
+
+exports.dump = (...args, obj) ->
+  util = require 'util'
+  console.log ...args, util.inspect obj, no, null
```

Comment:
We put the dots on the right in the CoffeeScript codebase; but besides, I don’t think we want debugging helpers in the codebase.